docs/competition/event/incident-management.md

```diff
@@ -0,0 +1,85 @@
+# SR2019 Incident Management Procedure
+
+This page is primarily for internal use. If you are a team competing in Student Robotics, you probably want our main website.
+
+This page details the procedures to be following in the event of an incident occuring during the competition. Incidents are split into 2 types; minor and major.
+
+What constitutes an incident is rather unpredictable, and as such is not fully defined. Common sense should prevail in determining the nature and severity of the incident.
+
+This procedure is superceded by any requirements of the venue, e.g. in the event of a fire follow the venues' fire evacuation procedure.
+
+# Minor Incidents
+
+A minor incident is anything that needs some kind of response but has minimal impact on the event.
+
+Examples may be;
+
+* A competitor has twisted their ankle on a stairwell,
+* Infrastructure cabling has become damaged and poses a hazard.
+
+Procedure here is simple;
+
+* Notify relevant coordinator,
+* Follow their instructions
+
+# Major Incidents
+
+A major incident is anything that causes a significant impact on the event as a whole.
+
+Examples might include;
+
+* Accident requiring significant first aid,
+* Battery failure renders access to an area unsafe
+
+As with minor incidents, the initial reaction should be to notify the relevant coordinator and follow their instructions. If first aid is required the First Aid Provider should __always__ be notified first.
+
+## Stopping the competition
+
+A major inicident may call for the competition to be stopped. This may be at the end of a current match or in the middle of a match.
+
+Due to the impact of stopping the competition, ability to do so is restricted to the following roles:
+
+* Health and Safety Cooordinator,
+* Event Coordinator,
+* Head Judge,
+* Majority decision from Trustees.
+
+Stopping the competition will be indicated by a call over the radio of "This is _role_; close the arena."
+
+At this point the following should occur;
+
+* Matches stop,
+* Power tools area to be vacated,
+* Battery charging to be turned off,
+* Competitors, Team Leaders, and members of the public should be asked to remain in their current location,
+* Competitors to be asked to stop testing on robots.
+
+The aim of these is to prevent any other issues occuring whilst the current incident is being dealt with, and to prevent attendees from getting in the way of the handling the incident.
+
+## Shift in roles
```

Comment:
Should we document this information in each of those roles?

Reply:
Potentially, though really all volunteers need to be familiar with this particular process, as if enacted it impacts everyone.

Reply:
Thinking about it, any significantly affected roles should probably just have a comment saying to check their responsibilities under this. That way we have a single source of truth should an incident ever arise, rather than any confusion from this differing from the other documents.